app/static/js/studentMinorPage.js

```diff
@@ -1,28 +1,56 @@
 $(document).ready(function(){
-    $("#expressInterest").on("click", function() {
+    
+
+    $("#summerExperienceSave").click(function(){
+      let data = {'summerExperience': $("#summerExperience").val(),
+                  'selectedSummerTerm': $("#summerExperienceTerm").find(":selected").text()}
       let username = $("#username").val()
-      let data = {"username":username}
       $.ajax({
-          url: "/cceMinor/"+username+"/indicateInterest",
-          type: "POST",
-          data: data,
-          success: function(s) {
-            console.log(s)
+        type: "POST", 
+        url: "/cceMinor/"+username+"/addSummerExperience",
+        data: data,
+        success: function(s){
+          msgToast("Success!", "Summer Experience successfully added!")
```

Comment:
Summer Experience is capitalized in this toast message, but not the error message.